### PR TITLE
More command line args to redex

### PR DESCRIPTION
**Pull request by justinjhendrick**
Also update the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +23,28 @@ buildscript {
...
+    keepFile = 'keep.txt'
+    jarFiles = ['lib1.jar', 'lib2.jar']
+    otherArgs = '' // any other command line options to `redex`
+    // see `redex --help` for details on these arguments
+}
+
 ```
-If you do not set the passes, all passes will be run by default. Sometimes you may not to run all optimisation passes, for example some appear to break when optimising kotlin code.
+If you do not set the passes or the config file, the default set of passes will
+be run. Sometimes you may not to run all optimisation passes, for example some
```
Comment (timmutton) [inline README.md]: May not want

Comment (justinjhendrick) [reply README.md]: How would you prefer it to be written? I changed this because the old version isn't quite true anymore.

Comment (timmutton) [reply README.md]: Sorry, meant to say that "Sometimes you may not to run all" should be changed to "Sometimes you may not want to run all"

Comment (justinjhendrick) [reply README.md]: Ah. Yeah. Of course. Good catch

**Comment by justinjhendrick** [comment]
Thanks!

**Pull request merged**